src/components/tooltip/Marker.js

```diff
@@ -629,6 +636,18 @@ function getMarkerDetails(
           <TooltipDetails>
             <TooltipDetail label="Type">{data.messageType}</TooltipDetail>
             <TooltipDetail label="Sync">{data.sync.toString()}</TooltipDetail>
+            <TooltipDetail label="Send Thread Latency">
+              {_maybeFormatDuration(data.startTime, data.sendStartTime)}
```

Comment:
I waffled back and forth on this a bit myself, but ended up with this way because I found it easier to understand the data with the "unknown" strings:

1. Since each of the 4 timespans have very different meanings (and conveying those meanings well with a short phrase each is hard), being able to remember "the Nth timespan means _____" helped me understand the measurements for a particular IPC message more quickly
2. Having an "unknown" value in the UI is also potentially helpful if someone reading the profile is expecting all 4 values to be known (similarly to (1), I find it easier to say "oh, the last timespan says 'unknown'; I must not be collecting markers on the recipient's I/O thread").

That said, I wouldn't be *too* upset if the "unknown" fields went away. (Also, in the future, I plan to provide a better high-level UI summarizing this data so that people reading the profile don't have to drill down into every single IPC marker to see the latencies. That might reduce the value of the "unknown" fields, since you'd be able to get this information in a better way...)